packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -254,6 +256,9 @@ try {
 		// Randomize the WordPress secrets
 		await defineWpConfigConsts(php, {
 			consts: {
+				WP_DEBUG: true,
+				WP_DEBUG_LOG: true,
+				WP_DEBUG_DISPLAY: false,
```

Comment:
Does this combination of settings only enable logging? Or does it also have other consequences?

Reply:
It only enables logging. `WP_DEBUG_DISPLAY: false` prevents printing of logs into the content.